packages/entity-list/src/components/AdminSearchForm/AdminSearchForm.js

```diff
@@ -60,6 +60,11 @@ const AdminSearchForm = ({resetSearch, intl, searchFilters}) => {
         onClick={resetSearch}
         title={msg('client.entity-list.reset')}
       />
+      <BallMenu buttonProps={{icon: 'ellipsis-h'}}>
```

Comment:
I tried out "-v", but since there's other buttons aligned horizontally to this one it bother me quite a bit. Might be personal taste though. Leave it to Yilmaz to decide?